modules/cannarithm/include/opencv2/acl_stream_accessor.hpp

```diff
@@ -0,0 +1,40 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_CANN_STREAM_ACCESSOR_HPP
+#define OPENCV_CANN_STREAM_ACCESSOR_HPP
+
+#include <acl/acl.h>
```

Comment:
BTW, this is platform-specific include in public header.

Some assumptions should be applied on the User side (like properly configured build environment, including headers paths and defines).

Reply:
I did some code refactoring, <acl/acl.h> only defined in cann_call.hpp, other files can't call acl functions directly.
In this commit(https://github.com/opencv/opencv/pull/22634), OpenCVFindCANN.cmake has already introduced, and cmake will set headers and libraries correctly.